MatrixSDK/Data/MXUser.m

```diff
@@ -112,7 +116,9 @@ - (void)updateWithPresenceEvent:(MXEvent*)presenceEvent inMatrixSession:(MXSessi
             self.avatarUrl = nil;
         }
     }
-    
+    if (presenceEvent.originServerTs > latestUpdateTS) {
+        latestUpdateTS = presenceEvent.originServerTs;
```

Comment:
Can you do the update only when we update the `avatarUrl` from `presenceContent`?